e2e/testsuite/testconfig.go

```diff
@@ -270,6 +276,14 @@ func (tc TestConfig) GetChainNumFullNodes(idx int) int {
 	return 0
 }
 
+// GetChainID returns the chain-id for i. Assumes indexs are correc.
```

Comment:
grammar nit police :D Sorry ^^
```suggestion
// GetChainID returns the chain-id for i. Assumes indicies are correct.
```

Reply:
Addressed here: [48aec70](https://github.com/cosmos/ibc-go/pull/8371/commits/48aec70a347409811fa073bb919c63c881ac0fe3)